app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :user, counter_cache: :posts_counter
+  has_many :comments
+  has_many :likes
+
+  def recent_comments
+    comments.limit(5).order(created_at: :desc)
+  end
+end
```

Comment:
Optional - It would look really nice if you used a Rails Active Callbacks - `after_save` to call the `#update_post_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method private. its optional and a cool feature to input